charts/opentelemetry-operator/values.yaml

```diff
@@ -181,6 +180,18 @@ admissionWebhooks:
     certificateAnnotations: {}
     issuerAnnotations: {}
 
+  ## Enable certificate auto generation if certManager is not enabled.
+  ## If true, Helm will automatically create self-signd cert and secret for you.
+  autoGenerateCert: true
```

Comment:
It doesn't feel right for `autoGenerateCert` and `certManager.enabled` to both be true by default.  I know it is safe because these values are only used when `certManager.enabled=false`, but if I was a user reading through this values.yaml file I think it would be confusing.

Can we add more comments in the values.yaml value explaining how these 2 fields interact?

Reply:
@TylerHelmuth updated. please let me know if you think that isn't clear and I'll try to add more detail. thanks

Reply:
This is great, thank you.